tests/bench/bench_util.go

```diff
@@ -43,26 +41,12 @@ var (
 func init() {
 	logging.SetConfig(logging.Config{Level: logging.NewLogLevelOption(logging.Error)})
 
-	// create a consistent seed value for the random package
-	// so we don't have random fluctuations between runs
-	// (specifically thinking about the fixture generation stuff)
-	seed := hashToInt64("https://xkcd.com/221/")
-	rand.Seed(seed)
```

Comment:
@jsimnz Do you see value in having a consistent seed and passing it the place where the random generation happens (as now it's not a global thing but a local object we would have to pass around with an initiated seed).

Reply:
Made issue for tracking: https://github.com/sourcenetwork/defradb/issues/1690